R/unspecified.R

```diff
@@ -55,76 +55,39 @@ vec_type_finalise.vctrs_unspecified <- function(x) {
 
 #' @rdname unspecified
 #' @export vec_type2.vctrs_unspecified
-#' @method vec_type2 vctrs_unspecified
 #' @export
-vec_type2.vctrs_unspecified <- function(x, y) UseMethod("vec_type2.vctrs_unspecified", y)
-#' @method vec_type2.vctrs_unspecified default
-#' @export
-vec_type2.vctrs_unspecified.default <- function(x, y) y
-#' @method vec_type2.vctrs_unspecified vctrs_unspecified
-#' @export
-vec_type2.vctrs_unspecified.vctrs_unspecified <- function(x, y) x
+vec_type2.vctrs_unspecified <- function(x, y) vec_slice(y, 0L)
```

Comment:
I think these should all call `vec_type()` instead of `vec_slice(y, 0L)`

Reply:
Changed two more instances of the same pattern.